client/reader/components/favicon/index.js

```diff
@@ -7,7 +7,7 @@ function Favicon( props ) {
 
 	// if loading error or missing icon show W Gridicon
 	if ( hasError || site.site_icon === null ) {
-		return <Gridicon icon="globe" size={ 18 } className={ props.className } />;
+		return <Gridicon icon="globe" size={ size } className={ props.className } />;
```

Comment:
Nice 👍 We seem to always provide `18` as a size in all instances of this component anyway.